templates/flasher.lua

```diff
@@ -1565,4 +1566,36 @@ function M.identify(tPlugin, aAttr, fnCallbackProgress, fnCallbackMessage)
 	return ulValue == 0
 end
 
+
+--------------------------------------------------------------------------
+-- Function to reset a netX through the flasher binary by
+-- triggering a watchdog reset
+--------------------------------------------------------------------------
+function M.reset(tPlugin, aAttr, fnCallbackProgress, fnCallbackMessage)
+	local romloader = require("romloader")
+	local iChipType = tPlugin:GetChiptyp()
+	local RESET_NETX_ENABLE_ALL = false  -- Allows use of reset_netx with all chip types
+
+	-- Only netX90s are officially supported. All other chips require activating the RESET_NETX_ENABLE_ALL bool
+	-- List of netX types in romloader repo -> romloader_def.h
+	if iChipType == romloader.ROMLOADER_CHIPTYP_NETX90
+	or iChipType == romloader.ROMLOADER_CHIPTYP_NETX90_MPW
+	or iChipType == romloader.ROMLOADER_CHIPTYP_NETX90B
+	or iChipType == romloader.ROMLOADER_CHIPTYP_NETX90C
+	or iChipType == romloader.ROMLOADER_CHIPTYP_NETX90D
+	or RESET_NETX_ENABLE_ALL
+	then
+		print("Resetting On-Chip")
+		local aulParameter =
+		{
+			OPERATION_MODE_Reset,                          -- operation mode: reset
+		}
+		local ulValue = callFlasher(tPlugin, aAttr, aulParameter, fnCallbackMessage, fnCallbackProgress)
+		return ulValue == 0
+	else
+		print("Error: This netX type does not support the reset_netx command")
+		return 0==1
```

Comment:
Is there a reason for using this instead of 'return false'?

Reply:
Changed it to false